### PR TITLE
phpunit 7.0.1

### PR DESCRIPTION
**Pull request by SMillerDev**
- [ ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew-php/pulls) for the same formula update/change?
- [x] Have you built your formula locally prior to submission with `brew install <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [x] Does your submission pass `brew audit --strict --online <formula>` (after doing `brew install <formula>`)?

-----
Removed the bottle block as it's pretty useless here. I also removed the now deprecate 5.7 version. We can also choose to keep it for a little longer with the latest supported version, which still has a PR open.

### CODE REVIEW

**Comment by alcaeus** [comment]
> We can also choose to keep it for a little longer with the latest supported version

Nah, kill it with fire 🔥

File: Formula/phpunit.rb [hunk 1]
```diff
@@ -4,15 +4,8 @@ class Phpunit < AbstractPhpPhar
   init
   desc "Programmer-oriented testing framework for PHP"
   homepage "https://phpunit.de"
-  url "https://phar.phpunit.de/phpunit-6.5.5.phar"
-  sha256 "7b871b0726d40dc4d783c08f4051b400615b0d85f0a5e26c86b9a75be47c8da1"
-
-  bottle do
-    cellar :any_skip_relocation
```
Comment (javian) [inline Formula/phpunit.rb]: You shouldn't need to touch this. It doesn't need to be updated but just so you know =)

Comment (SMillerDev) [reply Formula/phpunit.rb]: > Removed the bottle block as it's pretty useless here.
From the description. I know I don't have to, it was a conscious choice.

Comment (javian) [reply Formula/phpunit.rb]: Then we should add the bottle unneeded statement so it doesn't get added back =)

**Pull request closed without merging**